openedx/adg/lms/applications/models.py

```diff
@@ -195,14 +232,14 @@ class Meta:
         app_label = 'applications'
 
     def __str__(self):
-        return 'Education {id}, for {degree}'.format(id=self.id, degree=self.degree)
+        return ''
```

Comment:
In this case I guess we shouldn't override the method. Applied on other places as well.

Reply:
We have to, otherwise this is rendered with each education entry in application review page for admin:

`Education: Education object (1)`